### DISCUSSION TITLE
Buying a Plugin

### DISCUSSION THREAD

**Discussion by groundbreakingmc** [Zrips/CMI#7698] [Q&A]
I want to buy a plugin and would like to ask a few questions before buying.

1. My first question is: If I buy a plugin can I put it on all my servers?  (I have Proxy, Lobby, Grief, and Anarchy) Do I need to install a plugin for Grief Server and Anarchy, do I have to buy a plugin for each server?

2. After purchasing a plugin from Spigotmc, will I be able to download every plugin update?

3. If for some unknown reason the plugin does not work, can I get support or get a refund?

**Comment by Zrips** [comment]
One purchase for all of your servers.
Updates are free.
Support is provided if you have issues, mainly in a form of creating a ticket which will get addressed or you can visit Discord to get more of a live help out.

**Comment by CloudeLecaw** [comment]
https://www.zrips.net/faq/  you find all answers there in this faq


> Can I run a premium plugin on two servers or even twenty, if they are all behind BungeeCord?
>
> Yes, if you have a network of servers behind a proxy. You can use the same JAR on each server behind that proxy.
> 
> If the servers are yours, then one license is enough.


> 
> Will plugin x get more expensive in the future?
>
> If I buy the plugin for 1.19.2 and 1.20 comes out, do I have to buy it again?
> No, you do not have to purchase an upgrade license or a new license.
> 
> Getting a license for a premium plugin gives you access to future updates. (When they are ready and as long as the plugin gets updated.) 


> 
> Can I get a refund, the plugin isn't what I expected?
>
> We cannot offer any refunds (no refunds).
> 
> Please review the features before considering a purchase, and avoid purchasing what you don’t want to have.
> 


> Can I try a premium plugins out before buying, I don't want to waste money if I can't refund it?
>
> There’s no tryware-, shareware-, or freeware version of the premium plugins.
> 
> Use reviews from people who own a premium plugin to get an idea, or if you actually need it, and if it satisfies your needs before purchasing.
> 
> Side note: There are a few Zrips plugins that have their repositories on Github, CMI is not one of them by the way


![Screenshot_2023-03-24-06-17-37-735_com android chrome-edit](https://user-images.githubusercontent.com/12080017/227431015-2f7c90cc-9ff2-4997-8f61-9b9c80300b3f.jpg)
